src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,228 @@
+package duke;
+
+import duke.exceptions.DukeException;
+import duke.exceptions.ExceptionChecker;
+import duke.task.Task;
+import duke.task.TaskManager;
+import duke.util.InputParser;
+import java.util.Scanner;
+
+public class Duke {
```

Comment:
This class seems to be doing too many things. See if you can separate it out to different classes?